DEVELOP.md

```diff
@@ -0,0 +1,485 @@
+- [Command Line Interface](https://daffodil.apache.org/cli/)
+- [Community](https://daffodil.apache.org/community/)
+- [Configuration](https://daffodil.apache.org/configuration/)
+- [DFDL Extensions](https://daffodil.apache.org/dfdl-extensions/)
+- [Daffodil Java API](https://daffodil.apache.org/docs/latest/javadoc/)
+- [Daffodil Scala API](https://daffodil.apache.org/docs/latest/scaladoc/)
+- [Daffodil and the DFDL Infoset](https://daffodil.apache.org/infoset/)
+- [Data Format Description Language v1.0 Specification](https://daffodil.apache.org/docs/dfdl/)
+- [Eclipse Configuration](https://daffodil.apache.org/eclipse-configuration/)
+- [Examples](https://daffodil.apache.org/examples/)
+- [Frequently Asked Questions](https://daffodil.apache.org/faq/)
+- [Getting Started](https://daffodil.apache.org/getting-started/)
+- [Hidden Groups](https://daffodil.apache.org/dev/design-notes/hidden-groups/)
+- [Home](https://daffodil.apache.org/)
+- [Infoset Inputters and Runtime 1 Streaming Unparser & NextElementResolver](https://daffodil.apache.org/dev/design-notes/infoset-inputter-streaming-unparser/)
+- [Interactive Debugger](https://daffodil.apache.org/debugger/)
+- [Namespace Binding Minimization](https://daffodil.apache.org/dev/design-notes/namespace-binding-minimization/)
+- [People](https://daffodil.apache.org/people/)
+- [Releases](https://daffodil.apache.org/releases/)
+- [Runtime2 ToDos](https://daffodil.apache.org/dev/design-notes/runtime2-todos/)
+- [Standard DFDL Schema Project Layout](https://daffodil.apache.org/dfdl-layout/)
+- [Term Sharing in the Schema Compiler](https://daffodil.apache.org/dev/design-notes/term-sharing-in-schema-compiler/)
+- [Test Data Markup Language](https://daffodil.apache.org/tdml/)
+- [Understanding dfdl:bitOrder](https://daffodil.apache.org/tutorials/bitorder.tutorial.tdml.xml)
+- [Unsupported Features and Errata](https://daffodil.apache.org/unsupported/)
+- [VS Code Extension](https://daffodil.apache.org/vscode/)
+- [design notes folder](https://daffodil.apache.org/dev/)
+
+### Tests
+
+The majority of Daffodil tests are written inside TDML files with
+corresponding Scala unit tests which allow your SBT or IDE to run
+these tests within the TDML files.  Daffodil also has some pure Scala
+unit tests for testing smaller parts of Daffodil at a lower level.
+Both kinds of tests are important, although new Daffodil tests tend to
+be written in TDML and focus on testing issues with Daffodil's
+processing of DFDL schemas and parsing data or unparsing data.
+
+TDML tests are helpful for Daffodil developers because they allow a
+developer to write a TDML file with a schema, a test case, and test
+data which isolates a complex data format issue so the developer can
+work on that issue more easily.  The developer is free to decide
+whether to define the schema and test data inline in the TDML file or
+leave the schema and test data in separate files which the TDML file
+reads.
+
```

Comment:
Is it worth mentioning how to run these tests with sbt, especially since integration tests aren't run by default?

Reply:
Yes, I've added commands showing how to run the tests.